src/app/components/Footer.tsx

```diff
@@ -39,10 +38,7 @@ export const Footer = () => {
 
     let payload = {}
 
-    if (
-      appState?.appState?.settings?.bannerImage?.url ===
-      '/images/default_banner.png'
-    ) {
+    if (appState?.appState?.bannerImgUrl === '/images/default_banner.png') {
```

Comment:
this image url is being used in many places, so let's create a constant

Reply:
@saugatmaharjan changes made in the next commit